packages/api-contracts/ai-scan-api.yaml

```diff
@@ -0,0 +1,232 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+openapi: 3.0.0
+info:
+    version: 1.0.0
+    title: Accessibility Insight Service – Scan API
+paths:
+    /scans:
+        post:
+            summary: Create accessibility scan requests
+            operationId: createScans
+            parameters:
+                - name: api-version
+                  in: path
+                  required: true
+                  description: The client API version
+                  schema:
+                      type: string
+                  example: '1.0'
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/ScanRunRequests'
+            responses:
+                '201':
+                    description: Created
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ScanRunResponses'
+    /scans/{scanid}:
+        get:
+            summary: Get accessibility scan result
+            operationId: getScan
+            parameters:
+                - name: scanid
+                  in: path
+                  required: true
+                  description: The scan request ID
+                  schema:
+                      type: string
+                - name: api-version
+                  in: path
+                  required: true
+                  description: The client API version
+                  schema:
+                      type: string
+                  example: '1.0'
+            responses:
+                '200':
+                    description: OK
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/ScanResultResponse'
+    /scans/$batch:
+        post:
+            summary: Get accessibility scan results
+            operationId: getScanBatch
+            parameters:
+                - name: api-version
+                  in: path
+                  required: true
+                  description: The client API version
+                  schema:
+                      type: string
+                  example: '1.0'
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/ScanRuns'
+            responses:
```

Comment:
Should we also add error responses, maybe 404 / 410 here so that error contracts are also defined?

Here and elsewhere as well if needed.

Reply:
We are going to use plain HTTP 404 response. No need to define special contract for now.